code/modules/power/smes.dm

```diff
@@ -348,10 +348,9 @@
 	return 1
 
 /obj/machinery/power/smes/ui_interact(mob/user, ui_key = "main", var/datum/nanoui/ui = null, var/force_open = 1)
-
```

Comment:
Mhmmm, random loss of lines in random files.

Reply:
that would be @rickmund